src/rpcserver.h

```diff
@@ -59,6 +59,22 @@ class CRPCTable
 {
 private:
     std::map<std::string, const CRPCCommand*> mapCommands;
+    static constexpr const char* DEPRECATED_RPCS[] {
+            "debug",
+            "debug10",
+            "execute" ,
+            "getaccount",
+            "getaccountaddress",
+            "getaddressesbyaccount",
+            "getreceivedbyaccount",
+            "listaccounts",
+            "listreceivedbyaccount",
+            "list",
+            "move",
+            "setaccount",
+            "vote",
+    };
+
```

Comment:
Let's define this in rpcserver.cpp to reduce the header bloat. The rpcserver.h header is `#include`d into a lot of files still.

Reply:
Done.